package.json

```diff
@@ -75,6 +76,8 @@
     "node-fetch": "^2.6.7",
     "nodemailer": "^6.4.16",
     "nodemailer-sendgrid": "^1.0.3",
+    "redis": "^4.6.13",
+    "redis-clustr": "^1.7.0",
```

Comment:
I think redis-clustr can be removed.

Reply:
[6402f52](https://github.com/pagopa/io-functions-service-messages/pull/57/commits/6402f520351a9cc0e65f1b6137d66dc69efcdce0)